label_studio/users/api.py

```diff
@@ -93,7 +93,7 @@
 class UserAPI(viewsets.ModelViewSet):
     serializer_class = UserSerializer
     permission_required = ViewClassPermission(
-        GET=all_permissions.organizations_view,
+        GET=all_permissions.organizations_change,
         PUT=all_permissions.organizations_change,
         POST=all_permissions.organizations_change,
         PATCH=all_permissions.organizations_view,
```

Comment:
Something I noticed while looking at this, though not exactly related to your change - should this perm be `organizations_change`?

Reply:
No, because users should be able to change their own first and last name, etc. 
We have queryset restrictions, and editing of all users is possible only for administrators and org owners.